apple/internal/macos_rules.bzl

```diff
@@ -2174,6 +2189,7 @@ def _macos_dylib_impl(ctx):
         codesignopts = codesigning_support.codesignopts_from_rule_ctx(ctx),
         features = features,
         ipa_post_processor = None,
+        locales_to_include = ctx.attr.locales_to_include,
```

Comment:
dylib doesn't create a bundle, does it? Generally any rule that doesn't produce a bundle shouldn't have it.

Reply:
It doesn't; I only included it as potentially lproj files _could_ get stripped in `process` if the rule also supports `allows_locale_trimming` in `rule_support.bzl`? Is it worth preventing that?

As for bundles, am I correct in assuming all bundles ultimately call `resources_partial`? So i can make it an attribute only for rules that call into that? It's the only other callsite aside from `process`

Reply:
If `bundle_post_process_and_sign = False,` then `_bundle_partial_outputs_files` is never called. But in general, I would only add this to ones where `bundle_post_process_and_sign = True` and `allows_locale_trimming = True`.